iron-ajax.html

```diff
@@ -353,13 +353,15 @@
         contentType = 'application/x-www-form-urlencoded';
       }
       if (contentType) {
-        headers['Content-Type'] = contentType;
+        headers['content-type'] = contentType;
       }
       var header;
 
       if (this.headers instanceof Object) {
         for (header in this.headers) {
-          headers[header] = this.headers[header].toString();
+          // Normalize headers in lower case to make it easier for iron-request
+          // to reason about them.
+          headers[header.toLowerCase()] = this.headers[header].toString();
```

Comment:
`iron-request` isn't private, it's just not super widely used directly.

Here's a PR that moves the lowercasing loginc into iron-request, which better preserves the existing behavior while still fixing the bug that we were depending upon the capitalization of headers: https://github.com/PolymerElements/iron-ajax/pull/135